README.md

```diff
@@ -2,57 +2,79 @@
+
+## Manual
+Manual installation of the Lumigo tracer package requires the following steps:
+
+* Install the package: `pip install lumigo_tracer` 
+* Import the package in your Lambda code: `from lumigo_tracer import lumigo_tracer`
+* Wrap your function with the `@lumigo_tracer` or `@lumigo_tracer(token='XXX')` attributes.
+
+Once you've configured the package and added the tracer attribute, your code should resemble the following:
+
+```python
 @lumigo_tracer(token='XXX')
 def my_lambda(event, context):
     print('I can finally troubleshoot!')
 ```
 
 ## Configuration
-* You can turn on the debug logs by setting the environment variable `LUMIGO_DEBUG=true`
-* You can prevent lumigo from sending keys that answer specific regexes by defining `LUMIGO_SECRET_MASKING_REGEX=["regex1", "regex2"]`. By default, we use the default regexes `[".*pass.*", ".*key.*"]`. All the regexes are case-insensitive.
-* Similarly, you can prevent lumigo from sending the entire headers and body of specific domains using the environment variable `LUMIGO_DOMAINS_SCRUBBER=[".*secret.*"]` (give it a list which is a json parsable), or by specify the list of regexes with the key `domains_scrubber` in the tracer's decorator. By default, we will use `["secretsmanager\..*\.amazonaws\.com", "ssm\..*\.amazonaws\.com", "kms\..*\.amazonaws\.com"]`.
-* In case of need, there is a kill switch, that stops all the interventions of lumigo immediately, without changing the code. Simply add an environment variable `LUMIGO_SWITCH_OFF=true`.
+There are several configuration environment variables you can adjust to change how the package behaves:
+
+* `LUMIGO_DEBUG=true` - This environment variable controls debug logging. Set it to `true` to enable debug logging output
+* `LUMIGO_SECRET_MASKING_REGEX=["regex1", "regex2"]` - This is a JSON-formatted list of regular expressions that are applied to keys sent by Lumigo. If a key matches one of these regexes, it is not sent. All regular expressions are case-insensitive. The default list of regexes checked is  `[".*pass.*", ".*key.*"]`. All the regexes are case-insensitive.
+* `LUMIGO_DOMAINS_SCRUBBER=[".*secret.*"]` - This is a JSON-formatted list of regular expressions that can be used to prevent Lumigo from sending the entire headers and body of requests to specific domains. This can also be configured in the function decorator `@lumigo_tracer` using the parameter name `domains_scrubber`. The default list of regexes is `["secretsmanager\..*\.amazonaws\.com", "ssm\..*\.amazonaws\.com", "kms\..*\.amazonaws\.com"]`.
+* `LUMIGO_SWITCH_OFF=true` - This is a kill switch that stops all Lumigo-driven behavior when flipped, without needing to change the underlying code. Simply add this environment variable to your function's configuration, and the tracer will be disabled.
```

Comment:
Again these configurations are identical to nodejs, should use the same wording

Reply:
Copied and pasted!